pg_search/src/postgres/customscan/pdbscan/scan_state.rs

```diff
@@ -91,6 +90,7 @@ pub struct PdbScanState {
     pub var_attname_lookup: HashMap<(Varno, pg_sys::AttrNumber), String>,
     pub placeholder_targetlist: Option<*mut pg_sys::List>,
 
+    pub exec_method_type: ExecMethodType,
```

Comment:
Where is this consumed?

The general idea is that once the `ExecMethod` has been created, all of the fields of the `ExecMethodType` are accessible there. So it shouldn't be necessary to hold onto this?